housekeeper/cli/add.py

```diff
@@ -36,16 +36,20 @@ def bundle(context, name):
 @add.command('file')
 @click.option('-t', '--tag', 'tags', multiple=True, help='tag to associate the file by')
 @click.option('-a', '--archive', is_flag=True, help='mark file to be archived')
+@click.option('-v', '--version', help='creation datetime of the version [Y-m-d H:M:S]')
```

Comment:
We can validate the option even!

https://click.palletsprojects.com/en/6.x/options/#callbacks-for-validation